src/panels/lovelace/hui-root.ts

```diff
@@ -614,7 +617,13 @@ class HUIRoot extends LitElement {
     } else {
       if (viewConfig.panel && viewConfig.cards && viewConfig.cards.length > 0) {
         view = createCardElement(viewConfig.cards[0]);
-        view.isPanel = true;
+        view = document.createElement("div");
```

Comment:
you override the created view element above?

Reply:
Was supposed to be deleted... 🤦‍♂